pkg/ddc/alluxio/health_check.go

```diff
@@ -164,18 +166,17 @@ func (e *AlluxioEngine) checkWorkersHealthy() (err error) {
 		}
 
 		runtimeToUpdate := runtime.DeepCopy()
-
-		if workers.Status.NumberUnavailable > 0 ||
-			(workers.Status.DesiredNumberScheduled > 0 && workers.Status.NumberAvailable == 0) {
+		if (workers.Status.ReadyReplicas == 0 && *workers.Spec.Replicas > 0) ||
+			(*workers.Spec.Replicas > 0 && workers.Status.CurrentReplicas == 0) {
```

Comment:
What's the difference between `workers.Status.ReadyReplicas` and `workers.Status.CurrentReplicas`?

Or can we just think about `workers.Status.ReadyReplicas` when `workers.Spec.Replicas` is greater than 0

Can we define what a healthy state is?

This nested judgment is a bit complicated

Reply:
Nice advice! I will find a way to make it easier～